cognoml/data.py

```diff
@@ -101,9 +109,10 @@ def download_files(self):
         """
         version_to_url = self._version_to_url
         version = self._version
-        if version is None:
-            version = max(version_to_url.keys())
+        #if version is None:
```

Comment:
Why comment these lines out? Doesn't this break support for `version=None` returning the latest version?